exercise2.py

```diff
@@ -0,0 +1,57 @@
+class ArrayManipulation:
+	
+
+    def element_at(arr,index):
+        try:
+            if len(arr) < 1 :
+                raise AttributeError
+            if index > len(arr) or index < 0:
+                raise IndexError 
+            
+            return arr[index]
+        except IndexError:
+            return "Index value is out of bounds"
+        except AttributeError:
+            return "Array size should be greater than 0"
+
+    def inclusive_range(arr,start_pos,end_pos):
+        try: 
+            if len(arr) < 1 :
+                raise AttributeError
```

Comment:
can create a method that validates all inputs and reuse, follow DRY principle

Reply:
Done